test/unit/meson.build

```diff
@@ -0,0 +1,34 @@
+sources = files([
+    # Entrypoint
+    '../test.cpp',
+    # Unit tests
+    'FtlControlConnectionUnitTests.cpp',
+    'Rtp/ExtendedSequenceCounterTests.cpp',
+    # Project sources
+    '../../src/FtlControlConnection.cpp',
+    '../../src/FtlStream.cpp',
+    '../../src/Rtp/ExtendedSequenceCounter.cpp',
+    '../../src/Rtp/RtpPacket.cpp',
+])
+
+incdirs = include_directories(
+    '../../vendor/eventpp/include',
+    is_system: true,
+)
+
+deps = [
+    dependency('libssl'),
+    dependency('libcrypto'),
+    fmt_wrap.get_variable('fmt_dep'),
+    spdlog_wrap.get_variable('spdlog_dep'),
+    catch2_wrap.get_variable('catch2_dep'),
+]
+
+exe = executable(
```

Comment:
Not sure how I feel about having separate binaries for unit/functional/etc. test suites - does it make sense to build the individual suites as libraries that be linked into one big test executable?

Reply:
Thinking about it more, maybe it makes sense....